### PR TITLE
Issue546 sce flag

### PR DESCRIPTION
**Pull request by dhblum**
This is for #546.  It:
- implements test to check for problem
- implements fix
- updates release notes

### CODE REVIEW

**Comment by dhblum** [comment]
@haraldwalnum mind taking a review to make sure it solves the issue you reported?

File: testing/test_bestest_air.py [hunk 1]
```diff
@@ -43,6 +44,36 @@ def test_typical_cool_day(self):
...
+
+        '''
+
+        length = 86400*14
+        # Get current step
+        step_current = requests.get('{0}/step'.format(self.url)).json()['payload']
+        # Initialize test case scenario
+        requests.put('{0}/scenario'.format(self.url), json={'time_period':'peak_heat_day'})
+        # Set simulation step
+        requests.put('{0}/step'.format(self.url), json={'step':length}).json()['payload']
```
Comment (haraldwalnum) [inline testing/test_bestest_air.py:66]: Any reason why you get the json()['payload'] when it is not used?

File: testing/test_bestest_air.py [hunk 2]
```diff
@@ -43,6 +44,36 @@ def test_typical_cool_day(self):
...
+        # Simulation Loop
+        requests.post('{0}/advance'.format(self.url)).json()['payload']
```
Comment (haraldwalnum) [inline testing/test_bestest_air.py:68]: Same as above

**Review by haraldwalnum — COMMENTED** [review]
Looks good. Tested it locally, and it works for me.
Only a remark/question for the test.

Harald

File: testing/test_bestest_air.py (hunk 1 above)
Comment (dhblum) [reply testing/test_bestest_air.py:66]: No, that must have been left from a copy-paste.  I'll clean it up.

File: testing/test_bestest_air.py (hunk 2 above)
Comment (dhblum) [reply testing/test_bestest_air.py:68]: Same as above.

File: testing/test_bestest_air.py (hunk 1 above)
Comment (dhblum) [reply testing/test_bestest_air.py]: Resolved in https://github.com/ibpsa/project1-boptest/pull/566/commits/fa5bfedb137bbe73cb0266814c121b2d0fb0b007.

File: testing/test_bestest_air.py (hunk 2 above)
Comment (dhblum) [reply testing/test_bestest_air.py]: Resolved in https://github.com/ibpsa/project1-boptest/pull/566/commits/fa5bfedb137bbe73cb0266814c121b2d0fb0b007.

**Comment by dhblum** [comment]
Thanks @haraldwalnum for reviewing.  Will merge once tests pass.

**Pull request merged**